lua/bufferline.lua

```diff
@@ -268,6 +255,7 @@ function bufferline.setup(options)
     call dictwatcheradd(g:bufferline, '*', 'BufferlineOnOptionChanged')
   ]]
 
+  highlight.setup()
```

Comment:
I moved setup of highlight groups to setup of the plugin, since I figured if one does `:BarbarDisable` and then `:BarbarEnable` they wouldn't need the highlight groups setup again.